packages/document/docs/zh/config/options/options.mdx

```diff
@@ -68,269 +66,65 @@ new RsdoctorWebpackPlugin({
 
 </Tabs>
 
-## Options
+## 配置项
 
 **类型：** `Object`
 
 这个 `Options` 是 [RsdoctorWebpackPlugin](#rsdoctorwebpackplugin-%E6%8F%92%E4%BB%B6) 和 [RsdoctorRspackPlugin](#rsdoctorrspackplugin-%E6%8F%92%E4%BB%B6) 的配置项。它包含以下属性值：
 
-- [brief](#brief)
-- [disableClientServer](#disableclientserver)
-- [experiments](#experiments)
-- [features](#features)
-- [mode](#mode)
-- [output](#output)
-- [port](#port)
-- [supports](#supports)
-
-### brief
-
-<Badge text="Version: 0.4.0" type="warning" />
-
-- **类型：** [BriefType](#brieftype)
-- **可选：** `true`
-- **默认值：** `undefined`
+- [disableClientServer](./disableClientServer.mdx) - **客户端服务控制**：是否在分析完成后不自动打开 Rsdoctor 报告服务。
+- [experiments](./experiments.mdx) - **实验性功能配置**：启用 Rspack 插件的实验性功能，如原生插件性能优化。
+- [features](./features.mdx) - **特性开关配置**：控制 Rsdoctor 的各项分析功能，如 loader 分析、bundle 分析、规则检查等
+- [output](./output.mdx) - **输出相关配置**：控制报告的输出格式、目录等选项，包括 mode、options、reportCodeType 等
+- [port](./port.mdx) - **服务端口配置**：设置 Rsdoctor 开发服务器的端口号，用于本地开发时的报告预览
+- [supports](./supports.mdx) - **特性支持配置**：启用特定构建工具的兼容性支持，如 BannerPlugin 等
+- [brief](./brief.mdx) - **Brief 模式的配置**：控制简要模式报告的生成，包括 HTML 文件名和是否生成 JSON 数据文件
+- [mode](./mode.mdx) - **构建报告数据输出模式**：选择报告的输出模式，支持 normal（普通模式）和 brief（简要模式）
 
-Brief 模式更多配置，如下：
+### RsdoctorRspackPluginOptions
 
-- **reportHtmlName:** 配置 Brief 的 HTML 报告的名称，默认 `report-rsdoctor.html`。
-- **writeDataJson:** 默认 Brief 模式下将分析数据注入到 HTML 文件中，所以不会再产生构建分析数据。如果想要而外本地生成数据则需要配置 `writeDataJson: true`。
-
-#### briefType
+> RsdoctorWebpackPlugin 同 `Omit<RsdoctorRspackPluginOptions, 'experiments'>` 类型。
```

Comment:
i think it's better to add a `##` section instead of a quote here, hard to see it even after navigate by anchor.